tests/test_raw_image.py

```diff
@@ -15,6 +15,24 @@ def setUp(self):
             for y in range(self.height):
                 self.img.set_pixel(x, y, float(x + y * self.width))
 
+        self.test_gpu = [False]
+        if KB_HAVE_GPU:
+            self.test_gpu.append(True)
```

Comment:
This was my (probably misguided) attempt to avoid repeating the functions for the CPU and GPU versions. I agree that the `if` statements make it more confusing and brittle. I changed to use `@unittest.skipIf`